falcon/response.py

```diff
@@ -702,7 +706,7 @@ def delete_header(self, name):
         name = name.lower()
 
         if name == 'set-cookie':
-            raise HeaderNotSupported('This method cannot be used to remove cookies')
+            raise HeaderNotSupported('This method cannot be used to remove ookies')
```

Comment:
This was an accidental keystroke I believe.

Reply:
Fixed.